setup.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
```

Comment:
I made the setup.py file executable and added this shebang at the top.  It's so I can just do `./setup.py test` instead of the major overhead of `python setup.py test`...

Reply:
👍